libevmasm/CommonSubexpressionEliminator.cpp

```diff
@@ -219,7 +219,7 @@ void CSECodeGenerator::addDependencies(Id _c)
 {
 	if (m_classPositions.count(_c))
 		return; // it is already on the stack
-	if (m_neededBy.count(_c))
+	if (m_neededBy.find(_c) != m_neededBy.end())
```

Comment:
Can you use the `contains` helper?

Reply:
I am not sure of perfomance impact of such change.

Reply:
Good catch... we should actually specialize ``contains`` in ``libsolutil/CommonData.h`` for types that have ``.find`` similar to how it's done for ``findOffset``... but no need to do it in this PR, so I'd say just keeping it, is fine.

Reply:
Oh, I was hoping that std::find does it, but of course it can't.